engine/src/main/resources/assets/i18n/menu_id.lang

```diff
@@ -0,0 +1,255 @@
+{
+    "activate-module": "Aktifkan",
+    "add-server": "Tambah baru",
+    "add-server-tip": "Anda bisa menekan ENTER untuk mengcopy nama ke kotak alamat",
+    "advanced-usage": "Penggunaan tingkat lanjut!",
+    "animate-grass": "Animasikan Rumput",
+    "animate-water": "Animasikan Air",
+    "audio-settings": "Suara",
+    "audio-settings-title": "Pengaturan Suara",
+    "back": "Kembali",
+    "behavior-editor-assign": "masukan",
+    "behavior-editor-copy": "copy",
+    "behavior-editor-layout": "rancangan",
+    "behavior-editor-new": "baru",
+    "behavior-editor-paste": "paste",
+    "behavior-editor-remove": "hapus",
+    "binding-attack": "Serang",
+    "binding-autoMove-mode": "Toggle Gerak Otomatis",
+    "binding-backwards": "Mundur",
+    "binding-behavior-editor": "Toggle Editor Sifat",
+    "binding-chat": "Toggle Obrolan",
+    "binding-console": "Toggle Konsol",
+    "binding-crouch": "Merunduk / Turun",
+    "binding-crouch-mode": "Mode Merunduk",
+    "binding-decrease-view-distance": "Kurangi Jarak Pandang",
+    "binding-forwards": "Maju",
+    "binding-frob": "Pakai Taget",
+    "binding-hide-hud": "Sembunyikan HUD",
+    "binding-increase-view-distance": "Tambahkan Jarak Pandang",
+    "binding-jump": "Loncat / Naik",
+    "binding-left": "Serong Kiri",
+    "binding-nui-editor": "Editor NUI",
+    "binding-nui-skin-editor": "Editor Skin NUI",
+    "binding-pause": "Menu Permainan",
+    "binding-right": "Serong Kanan",
+    "binding-show-online-players": "Tunjukan Pemain Online",
+    "binding-toggle-speed-permanently": "Toggle Kecepatan Secara Permanen",
+    "binding-toggle-speed-temporarily": "Toggle Kecepatan Secara Sementara",
+    "binding-use-item": "Pakai Barang di Tangan",
+    "bloom-effect": "Bloom",
+    "bobbing": "Meangangguk",
+    "camera-blur": "Blur",
+    "camera-blur-max": "Maks",
+    "camera-blur-normal": "Normal",
```

Comment:
I think there _should_ be a translation for this. [3]

(Unless this is already the correct translation)